shell/pages/c/_cluster/apps/charts/index.vue

```diff
@@ -100,7 +101,13 @@ export default {
           }
         }
       ],
-      appCardsCache: {},
+      appCardsCache:      {},
+      selectedSortOption: 'featured',
+      sortOptions:        [
+        { value: 'featured', label: this.t('catalog.charts.sortBy.recommended') },
```

Comment:
think these values are used often enough (repeatedly in here and referenced also in the catalog store) to be an enum somewhere (probably in shell/types)